config/routes.rb

```diff
@@ -25,6 +25,10 @@
 
   get "/confirmation-letter/:id", to: "confirmation_letter#show", as: :confirmation_letter
 
+  # Renewal Letter
+
```

Comment:
```suggestion
```

Reply:
Resolved by https://github.com/DEFRA/waste-exemptions-back-office-ta/pull/357/commits/70006d2f56bf3dfdf1325e9a28ba478f2b310357 (had to make more changes for this to be consistent)